src/software/thunderscope/gl/layers/gl_world_layer.py

```diff
@@ -72,22 +73,22 @@ def __init__(
         self.point_in_scene_picked = None
 
         self.friendly_defense_area_graphic = GLRect(
-            parentItem=self, color=Colors.FIELD_LINE_COLOR, line_width=3
+            parentItem=self, color=Colors.FIELD_LINE_COLOR
         )
         self.enemy_defense_area_graphic = GLRect(
-            parentItem=self, color=Colors.FIELD_LINE_COLOR, line_width=3
+            parentItem=self, color=Colors.FIELD_LINE_COLOR
         )
         self.field_lines_graphic = GLRect(
-            parentItem=self, color=Colors.FIELD_LINE_COLOR, line_width=3
+            parentItem=self, color=Colors.FIELD_LINE_COLOR
         )
         self.field_outer_boundary_graphic = GLRect(
-            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR, line_width=3
+            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR
         )
         self.halfway_line_graphic = GLLinePlotItem(
-            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR
+            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR, width=3.0
         )
         self.goal_to_goal_line_graphic = GLLinePlotItem(
-            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR
+            parentItem=self, color=Colors.FIELD_LINE_LIGHTER_COLOR, width=3.0
```

Comment:
Similarly for all the line width? There are lots of 3.0s around the code now, can we have a constant for this so it's easily tunable?

Reply:
Done